config/vufind/channels.ini

```
@@ -179,52 +184,66 @@ maxValuesToSuggestPerField = 2
 displayPublicLists = true
 ; How many lists should we display initially? (The rest can be added on demand)
 initialListsToDisplay = 2
-; Number of results to include in each channel.
-channelSize = 6
+; Number of columns to show in each row
+itemsPerRow = 6 ; default 6
+; Number of rows to show (and add when 'load more' is clicked)
+rowsPerPage = 1 ; default 1
 
 ; This section contains default settings for the NewILSItems channel provider
 [provider.newilsitems]
-; Number of results to include in the channel.
-channelSize = 6
 ; Maximum age of results to return (in days)
 maxAge = 60
+; Number of columns to show in each row
+itemsPerRow = 6 ; default 6
+; Number of rows to show (and add when 'load more' is clicked)
+rowsPerPage = 1 ; default 1
 
 ; This section contains default settings for the NewSearchItems channel provider
 [provider.newsearchitems]
-; Number of results to include in the channel.
-channelSize = 6
 ; Maximum age of results to return (in days)
 maxAge = 60
 ; Sort order for search results.
 sort = "first_indexed desc"
+; Number of columns to show in each row
+itemsPerRow = 6 ; default 6
+; Number of rows to show (and add when 'load more' is clicked)
+rowsPerPage = 1 ; default 1
 
 ; This section contains default settings for the Random channel provider
-[provider.random]
-; Number of results to include in the channel.
-channelSize = 6
+[provider.random]\
```

Looks like a possible typo here:

```suggestion
[provider.random]
```